app/controllers/chapters_controller.rb

```diff
@@ -9,6 +9,9 @@ def index
 
   def show
     @chapter = Chapter.find(params[:id])
-    render :json => @chapter
+
```

Comment:
By the way, I think if you rename the file to `show.json.jbuilder`, it will automatically load that JSON file. 

Not sure what the best practice is. I do like how this is explicit!

Reply:
ChatGPT thinks it's best to stick with RoR convention of `show.json.jbuilder`. Should I do that?

Reply:
Theek sounds good